e2e/flags.go

```diff
@@ -83,6 +83,10 @@ var E2E = flag.Bool("e2e", false,
 var Stress = flag.Bool("stress", false,
 	"If true, run stress tests.")
 
+// Profiling enables running of profiling tests.
+var Profiling = flag.Bool("profiling", false,
```

Comment:
Do you intend to add a CI job to continuously run the profiling tests, or will this just be invoked manually?

Reply:
Manual. These tests are expensive and long and need cluster autoscaling to function. They also perform poorly on new clusters and often need to be run multiple times to cause the control plan to scale up sufficiently. They also eat a lot of quota and subnet IPs.

Reply:
Ack, SGTM.